test/test_cargo.py

```diff
@@ -120,7 +120,7 @@ def test_get_package_metadata(exclude):
             input_toml += f'{k} = "{v}"\n'
 
     assert (get_package_metadata(io.BytesIO(input_toml.encode("utf-8"))) ==
-            PackageMetadata(**data))  # type: ignore
+            PackageMetadata(**data, features={}))  # type: ignore
```

Comment:
Having to add `features={}` everywhere when the input didn't change looks wrong.